guides/index_lifecycle.md

```diff
@@ -0,0 +1,174 @@
+            }
+        },
+        "mappings": {
+            "properties": {
+                "title": { "type": "text" },
+                "year": { "type": "integer" }
+            }
+        }
+    })).send().await?;
+```
+
+When you create a new document in an index, OpenSearch will automatically create the index if it doesn't exist:
+
+```rust
+println!("{}", client.indices().exists(IndicesExistsParts::Index(&["burner"])).send().await?.json::<bool>().await?); // => false
+client
+  .index(IndexParts::Index("burner")).body(json!({ "lorem": "ipsum" })).send().await?;
+println!("{}", client.indices().exists(IndicesExistsParts::Index(&["burner"])).send().await?.json::<bool>().await?); // => true
+```
+
+### Update an Index
+
+You can update an index's settings and mappings by using the `indices.put_settings` and `indices.put_mapping` API actions.
+
+The following example updates the `movies` index's number of replicas to `0`:
+
+```rust
+client
+    .indices().put_settings(IndicesPutSettingsParts::Index(&["movies"])).body(json!({
+        "index": {
+            "number_of_replicas": 0
+        }
+    })).send().await?;
+```
+
+The following example updates the `movies` index's mappings to add a new field named `director`:
+
+```rust
+client
+    .indices().put_mapping(IndicesPutMappingParts::Index(&["movies"])).body(json!({
+        "properties": {
+            "director": { "type": "text" }
+        }
+    })).send().await?;
+```
+
+### Get Metadata for an Index
+
+Let's check if the index's settings and mappings have been updated by using the `indices.get` API action:
+
+```rust
+println!("{:#?}", client.indices().get(IndicesGetParts::Index(&["movies"])).send().await?.json::<serde_json::Value>().await?);
+```
+
+The response body contains the index's settings and mappings:
+
+```json
+{
+  "movies": {
+    "aliases": {},
+    "mappings": {
+      "properties": {
+        "title": { "type": "text" },
+        "year": { "type": "integer" },
+        "director": { "type": "text" }
+      }
+    },
+    "settings": {
+      "index": {
+        "creation_date": "1680297372024",
+        "number_of_shards": "2",
+        "number_of_replicas": "0",
+        "uuid": "FEDWXgmhSLyrCqWa8F_aiA",
+        "version": { "created": "136277827" },
+        "provided_name": "movies"
+      }
+    }
+  }
+}
+```
+
+### Delete an Index
+
+Let's delete the `movies` index by using the `indices.delete` API action:
+
+```rust
+client
+    .indices().delete(IndicesDeleteParts::Index(&["movies"])).send().await?;
+```
+
+We can also delete multiple indices at once:
+
+```rust
+client
+    .indices().delete(IndicesDeleteParts::Index(&["movies", "paintings", "burner"])).ignore_unavailable(true)
+    .send().await?;
+```
+
+_Note_: The ignore_unavailable method allows for error tracing in the server response. If the index does not exist when removing the index, it will be ignored and the send method will return a successful result.
```

Comment:
This note still references "error tracing" which is not what the `ignore_unavailable` parameter does